src/lib/cache.rs

```diff
@@ -38,13 +38,10 @@ impl RedisConnection {
         let mut conn = self.0.clone().get().unwrap();
         redis::pipe()
             .atomic()
-            .add_command(redis::cmd("SET"))
-            .arg(key)
-            .arg(value)
-            .ignore()
-            .add_command(redis::cmd("EXPIRE"))
+            .add_command(redis::cmd("SETEX"))
```

Comment:
Just FYI, Redis is thinking about deprecating `SETEX` as `SET` options replace SETEX:

![image](https://user-images.githubusercontent.com/24852502/63921953-0612e800-ca44-11e9-9ef8-e2408abe9aa1.png)

The right syntax would be `SET key value EX ttl`, as tried in Redis' command line:

```
redis> SET myotherkey "Hello" EX 10
"OK"
redis> TTL myotherkey
(integer) 6
```